packages/discord-bitflag/src/wrappers/applications.ts

```diff
@@ -0,0 +1,9 @@
+import { BitField } from "bitflag-js";
+import { ApplicationFlags } from "../flags/application.js";
+
+export class ApplicationFlagsBitField extends BitField {
+	static ALL = Object.values(ApplicationFlags).reduce(
```

Comment:
This could be simplified to just pass `Object.values(ApplicationFlags)` to `BitField.resolve()`, more of a minor thing, but might make the code a little more readable than the reduce (and same on all the other ones)

Reply:
Much better yeah